docs/common-problems.asciidoc

```diff
@@ -66,7 +68,7 @@ is coming from an origin not whitelisted in `apm-server.rum.allow_origins`. See
 [float]
 === HTTP 413: Request body too large
```

Comment:
should be more visible the fact that this only can happen for v1?

Reply:
I initially left out a note here because this `<<configuration-v1-api,`apm-server.max_unzipped_size`>>` links to a section that mentions this is v1 only. But after reading it over again, it doesn't hurt to also mention this in the `common-problems.asciidoc` file - so I've added it in.